README.md

```diff
@@ -38,14 +39,15 @@ This repository is a collaborative effort, so please feel free to make a pull re
 ### Tools
 
 - [Editor and tool support](ecosystem/Editor-and-tool-support.md): Editor plugins, build tools, and other development tools
-- [PureScript and NixOS](https://pr06lefs.wordpress.com/2015/01/11/get-started-with-purescript-on-nixos/): How to use PureScript with NixOS
+- [`easy-purescript-nix`](https://github.com/justinwoo/easy-purescript-nix): Set up PureScript and development environment with [Nix](https://nixos.org/)
 - [PSCi](guides/PSCi.md): An interactive development tool for PureScript
 
 ### Ecosystem
 
 - [Maintained Packages](ecosystem/Maintained-Packages.md)
 - [Style Guide](guides/Style-Guide.md)
 - [Alternate Backends](https://github.com/purescript/documentation/blob/master/ecosystem/Alternate-backends.md): PureScript can compile to other languages as well!
+- [Recommended tooling for PureScript in 2020](https://discourse.purescript.org/t/recommended-tooling-for-purescript-in-2020/1615)
```

Comment:
There's an attempt to do this for the 2019 thread in https://github.com/purescript/documentation/issues/274. Some of the considerations discussed there also apply to this PR. I'm personally in favor of linking to that thread, or even tracking its contents in this repo so it can more easily be continuously updated.

Reply:
I think this and the above reference to `easy-purescript-nix` can be removed once #343 is merged, as they will both be included in the "Editor and tool support" documentation.